lib/logstash-input-http_jars.rb

```diff
@@ -4,4 +4,4 @@
 require_jar('io.netty', 'netty-all', '4.1.18.Final')
 require_jar('io.netty', 'netty-tcnative-boringssl-static', '2.0.7.Final')
 require_jar('org.apache.logging.log4j', 'log4j-api', '2.6.2')
-require_jar('org.logstash.plugins.input.http', 'logstash-input-http', '4.0.0')
+require_jar('org.logstash.plugins.input.http', 'logstash-input-http', '3.1.0')
```

Comment:
accidental ?

Reply:
it was on purpose, this no longer is a breaking change PR, so I bumped only to 3.1.0 instead of 4.0.0